.storybook/themes/news.scss

```diff
@@ -1895,6 +1929,39 @@
 				masthead: (
 					display: flex,
 				),
+				medium-promo: (
+					components: (
+						attribution: (
+							display: block,
+						),
+						media-item: (
+							float: left,
+							margin: 0,
+							max-width: 33%,
+						),
+						heading: (
+							font-size: var(--global-font-size-11),
+							line-height: var(--global-line-height-7),
+						),
```

Comment:
```suggestion
```
These aren't needed with the heading-level-5 changes above.

Reply:
Sorry, got these but must not have pushed up the changes. 
These have been removed.